src/main/java/uk/gov/hmcts/ccd/domain/types/CaseDataValidator.java

```diff
@@ -145,13 +149,21 @@ private List<ValidationResult> validateSimpleField(final String fieldId,
         Optional<FieldValidator> validatorToExecute =
             fieldIdBasedValidator.or(() -> customTypeValidator).or(() -> baseTypeValidator);
 
-        return validatorToExecute.map(validator -> validator.validate(validationContext)
-                .stream()
-                .map(result ->
-                    new ValidationResult(result.getErrorMessage(), fieldIdPrefix + result.getFieldId()))
-                .collect(Collectors.toList()))
+        List<ValidationResult> validationResults;
+        try {
+            validationResults = validatorToExecute.map(validator -> validator.validate(validationContext)
+                    .stream()
+                    .map(result ->
+                        new ValidationResult(result.getErrorMessage(), fieldIdPrefix + result.getFieldId()))
+                    .collect(Collectors.toList()))
                 .orElseThrow(() ->
                     new RuntimeException("System error: No validator found for " + fieldType.getType()));
+        } catch (RuntimeException e) {
+            LOGGER.error("No validator found for field " + fieldIdPrefix + fieldId + ", type " + fieldType.getType());
+            throw e;
+        }
+
+        return validationResults;
```

Comment:
You may want to add you logging directly into **.orElseThrow** as below to reduce the change.
`.orElseThrow(() ->{
                    log.error("No validator found for field " + fieldIdPrefix + fieldId + ", type " + fieldType.getType());
                    return new RuntimeException("System error: No validator found for " + fieldType.getType());
                });`

Reply:
Agreed - will change orElseThrow() as suggested.